corehq/apps/users/views/__init__.py

```diff
@@ -1004,3 +1004,12 @@ def location_restriction_for_users(request, domain):
         project.location_restriction_for_users = json.loads(request.POST["restrict_users"])
     project.save()
     return HttpResponse()
+
+
+@require_POST
+@require_superuser
```

Comment:
Actually now that I think about this, is this endpoint hit by the mobile worker, or web user account? we pass in the couch id of the web user, so I imagine this will be hit with the mobile worker.

I think that this decorator will call all of these to fail.